docs/pipeline-components-and-applications/iglu/igluctl-2/index.md

```diff
@@ -19,7 +19,8 @@ Iglu provides a CLI application, called igluctl which allows you to perform most
     - `generate` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for Redshift and Postgres warehouses. Generate DDLs and migrations from set of JSON Schemas. If the schema is not evolved correctly and backward incompatible data is sent within transformer's aggregation window, loading would fail for all events.
     - `push` - push set of JSON Schemas from static registry to full-featured (Scala Registry for example) one
     - `pull` - pull set of JSON Schemas from registry to local folder
-    - `parquet-verify` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for parquet transformation (for loading into Databricks).  If the schema is not evolved correctly, the offending version will be stored in a separate column with `_recovered_` in its name.
+    - `verify-parquet` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for parquet transformation (for loading into Databricks). It reports the breaking schema versions.
+    - `verify-redshift` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for loading into Redshift. It reports the schema criterions which belongs to breaking schema versions.
```

Comment:
> schema criterions which belongs to breaking schema versions

Not sure I understand this part. Is there a chance we could avoid using the words “schema criterions”? I suspect very few people know what they mean

Reply:
`verify-redshift`'s output is like this:
```
iglu:com.snowplowanalytics.iglu/resolver-config/jsonschema/1-*-*
iglu:com.snowplowanalytics.snowplow/event_fingerprint_config/jsonschema/1-*-*
iglu:com.snowplowanalytics.snowplow.badrows/loader_runtime_error/jsonschema/1-*-*
```
We might say `It reports the breaking schema versions` instead even though it is not exactly true.

Reply:
Maybe “it reports the major schema versions within which schema evolution rules were broken”?